sim/warrior/arms/arms.go

```diff
@@ -39,28 +42,95 @@ func NewArmsWarrior(character *core.Character, options *proto.Player) *ArmsWarri
 		Options: armsOptions,
 	}
 
-	// rbo := core.RageBarOptions{
-	// 	StartingRage:   armsOptions.ClassOptions.StartingRage,
-	// 	RageMultiplier: core.TernaryFloat64(war.Talents.EndlessRage, 1.25, 1),
-	// }
-	// if mh := war.GetMHWeapon(); mh != nil {
-	// 	rbo.MHSwingSpeed = mh.SwingSpeed
-	// }
-	// if oh := war.GetOHWeapon(); oh != nil {
-	// 	rbo.OHSwingSpeed = oh.SwingSpeed
-	// }
+	rbo := core.RageBarOptions{
+		StartingRage: armsOptions.ClassOptions.StartingRage,
+		OnHitDealtRageGain: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult, rage float64) float64 {
+			return rage * 1.25 // Endless Rage is now part of Anger Management, now an Arms specialization passive
+		},
+	}
+	if mh := war.GetMHWeapon(); mh != nil {
+		rbo.MHSwingSpeed = mh.SwingSpeed
+	}
+	war.EnableRageBar(rbo)
 
-	// war.EnableRageBar(rbo)
-	// war.EnableAutoAttacks(war, core.AutoAttackOptions{
-	// 	MainHand:       war.WeaponFromMainHand(war.DefaultMeleeCritMultiplier()),
-	// 	OffHand:        war.WeaponFromOffHand(war.DefaultMeleeCritMultiplier()),
-	// 	AutoSwingMelee: true,
-	// 	ReplaceMHSwing: war.TryHSOrCleave,
-	// })
+	war.EnableAutoAttacks(war, core.AutoAttackOptions{
+		MainHand:       war.WeaponFromMainHand(war.DefaultMeleeCritMultiplier()),
+		AutoSwingMelee: true,
+	})
```

Comment:
Can't arms warrior dual wield anymore? The way you've changed this you have removed support for dual wielding completely

Reply:
Only Fury is able to dual wield now. See the wording of https://www.wowhead.com/cata/spell=23588/dual-wield-specialization

There's also a patch note about it here https://wowpedia.fandom.com/wiki/Dual_wield#Patch_changes

Reply:
Interesting. Did not know they flat out removed it. Leaving a note to self to limit the availability of equipping OH weapons in those specs that no longer have DW